examples/stm32/f3/stm32f3-discovery/flash/Makefile

```diff
@@ -0,0 +1,26 @@
+##
+## This file is part of the libopencm3 project.
+##
+## Copyright (C) 2009 Uwe Hermann <uwe@hermann-uwe.de>
```

Comment:
This is not true.  This is not a copy of another file, and Uwe didnt' write this file in 2009.  Either drop it completely, or use your own name here.  This comment applies throughout this PR, but I won't comment on it again.